### ISSUE TITLE
[CI] Build failing: Spec2-Core can not be loaded 

### ISSUE THREAD

**Issue by MarcusDenker** [guillep/pharo-core#16443]
**Bug description**
on the CI when checking PRs we get now:

```
NewUndeclaredWarning: SpPresenterSelectorPresenter>>inputModelPort (SpModelPort is Undeclared)
MetacelloNotification: Loaded -> Spec2-Core --- github://pharo-spec/Spec:Pharo12/src(57359c2)
Error: The following definitions had errors while loading.  Press Proceed to try to load them again (they may work on a second pass):
  SpRequestDialog
  SpRequestTextDialog


```

**Comment by Rinzwind** [comment]
This seems to have been fixed in [Spec commit 6186563ddba5](https://github.com/pharo-spec/Spec/commit/6186563ddba53af146466455b2cc65216236f522).

**Comment by MarcusDenker** [comment]
fixed !
